src/main/scala/Server.scala

```diff
@@ -42,16 +43,26 @@ object Server extends IOApp {
       _      <- pingTokenSync(configRef, httpClient)
     } yield ()
 
-  private def plexTokenSync(
+  private def plexRssSync(
       configRef: Ref[IO, Configuration],
-      httpClient: HttpClient,
-      firstRun: Boolean = true
+      httpClient: HttpClient
   ): IO[Unit] =
     for {
       config <- fetchLatestConfig(configRef)
-      _      <- PlexTokenSync.run(config, httpClient, firstRun)
+      _      <- PlexTokenSync.run(config, httpClient, runFullSync = false)
       _      <- IO.sleep(config.refreshInterval)
-      _      <- plexTokenSync(configRef, httpClient, firstRun = false)
+      _      <- plexRssSync(configRef, httpClient)
+    } yield ()
+
+  private def plexFullSync(
+      configRef: Ref[IO, Configuration],
+      httpClient: HttpClient
+  ): IO[Unit] =
+    for {
+      config <- fetchLatestConfig(configRef)
+      _      <- PlexTokenSync.run(config, httpClient, runFullSync = true)
+      _      <- IO.sleep(19.minutes)
```

Comment:
Forgot to add my reasoning for why 19 minutes specifically - it avoids accidental clustering (e.g. if a few servers in the world are set to restart at midnight every night, this gives enough randomness that they won't start hammering Plex every 20 mins)